library/network/src/modules/SuSEFirewall.rb

```diff
@@ -38,6 +38,9 @@ class SuSEFirewallClass < Module
 
     include Yast::Logger
 
+    YES = "yes"
+    NO  = "no"
```

Comment:
and also `@SETTINGS["FW_BOOT_FULL_INIT"] == INIT_MAPPING[true]` ?
that sounds strange